versions.tf

```
@@ -19,7 +19,7 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 3.53, < 5.0"
+      version = "<= 4.74, != 4.75.0"
```

It would be good to preserve the min constraint 
```suggestion
      version = ">= 3.53, <= 4.74, != 4.75.0"
```